### PR TITLE
Assert check

### LINKED ISSUE #54: Generate init_check_assert
For now it's missing.

Option to add in `Gir.toml` (besides the macros in `rt.rs`): `generate_safety_asserts=true`

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #54.

cc @sdroege @EPashkin

### CODE REVIEW

**Comment by sdroege** [comment]
cc @jsparber

File: src/rt.rs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+pub fn is_initialized() -> bool {
+    skip_assert_initialized!();
+    INITIALIZED.load(Ordering::Acquire)
+}
+
+/// Returns `true` if Sourceview has been initialized and this is the main thread.
+#[inline]
+pub fn is_initialized_main_thread() -> bool {
+    skip_assert_initialized!();
+    IS_MAIN_THREAD.with(|c| c.get())
```
Comment (sdroege) [inline src/rt.rs]: Both booleans are never actually set to `true` or am I missing something?

You basically need to ensure that `IS_MAIN_THREAD` is only `true` on the thread on which GTK was initialized, and `INITIALIZED` must be `true` once GTK is initialized.

Comment (GuillaumeGomez) [reply src/rt.rs]: Oups indeed! Fixing it.

**Comment by sdroege** [comment]
:+1: or just move those few lines into `lib.rs`

**Comment by GuillaumeGomez** [comment]
No, I like having a `rt` file around. :)

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks, 👍 too

**Pull request merged**